sdk/appconfiguration/Azure.Data.AppConfiguration/samples/README.md

```diff
@@ -0,0 +1,73 @@
+# Azure App Configuration client SDK samples
+
+This directory contains samples for the Azure App Configuration client SDK.  Each sample illustrates one or more capabilities of the library.
+
+These include:
+ - [Hello World](#hello-world): Create and delete a configuration setting.
+ - [Hello World Extended](#hello-world-extended): Asynchronously create, update and delete configuration settings with labels.
+ - [Set Read Only](#set-read-only): Make a configuration setting read only, and then return it to a read write state.
+ - [Read Revision History](#read-revision-history): Read the revision history of a configuration setting that has been changed.
+ - [Get Setting If Changed](#get-setting-if-changed): Save bandwidth by using a conditional request to retrieve a setting only if it is different from your local copy.
+ - [Update Setting If Unchanged](#update-setting-if-unchanged): Prevent lost updates by using optimistic concurrency to update a setting only if your local updates were applied to the same version as the resource in the configuration store.
+ - [Mock Client](#mock-client): Mock a client for testing using the [Moq library][moq].
+
+ ## Prerequisites
+ To run these samples, you will need:
+ - An [Azure subscription][azure_sub].
+ - A [Configuration Store][configuration_store].
+ - To set the environment variable ``"APPCONFIGURATION_CONNECTION_STRING"`` to the connection string for your configuration store.
```

Comment:
1. Just a single backtick for a code fragment.
2. Nit: I wouldn't put the variable in quotes after defining it as a code fragment. It's not like the variable would actually have quotes in the name, and the code fragment already sets it apart.